spiceaidocs/docs/data-accelerators/index.md

```diff
@@ -2,7 +2,6 @@
 title: 'Data Accelerators'
 sidebar_label: 'Data Accelerators'
 description: ''
-sidebar_position: 5
```

Comment:
is that expected?

![CleanShot 2024-05-14 at 08 32 02@2x](https://github.com/spiceai/docs/assets/827338/e0891ae5-7d3f-46fe-8a70-2d60565f1765)